case_mapping/uco/observable.py

```diff
@@ -415,6 +425,52 @@ def __init__(self, *args: Any, has_changed=None, state=None, **kwargs: Any) -> N
         self._bool_vars(**{"uco-observable:hasChanged": has_changed})
 
 
+class FacetApplication(FacetEntity):
```

Comment:
Review note: This class definition needed to move below `ObservableObject`'s definition for a type-signature reference.